core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -106,27 +107,22 @@
 import nl.nn.adapterframework.util.XmlUtils;
 
 /**
- * The receiver is the trigger and central communicator for the framework.
- * <br/>
- * The main responsibilities are:
- * <ul>
- *    <li>receiving messages</li>
- *    <li>for asynchronous receivers (which have a separate sender):<br/>
- *            <ul><li>initializing ISender objects</li>
- *                <li>stopping ISender objects</li>
- *                <li>sending the message with the ISender object</li>
- *            </ul>
- *    <li>synchronous receivers give the result directly</li>
- *    <li>take care of connection, sessions etc. to startup and shutdown</li>
- * </ul>
- * Listeners call the IAdapter.processMessage(String correlationID,String message)
- * to do the actual work, which returns a <code>{@link PipeLineResult}</code>. The receiver
- * may observe the status in the <code>{@link PipeLineResult}</code> to perform committing
- * requests.
- *
- *
- * </p>
- * <p><b>Transaction control</b><br>
+ * Specific input source for the messages of a specific {@link Adapter}. Details about the input
+ * source are determined by the listener nested in the receiver. The added value of
```

Comment:
Ik vind 'added value' hier een beetje een rare formulering. Het is niet dat het ook zonder een Receiver kan.

Reply:
Ik voer de receiver nu in de eerste zin op als wrapper van zijn listener. Dan loopt het verhaal een stuk beter.